test/e2e/kibishii_utils.go

```diff
@@ -88,19 +83,14 @@ func verifyData(ctx context.Context, namespace string, levels int, filesPerLevel
 	return nil
 }
 
-// runKibishiiTests runs kibishii tests on the provider.
-func runKibishiiTests(client testClient, providerName, veleroCLI, veleroNamespace, backupName, restoreName, backupLocation string,
-	useVolumeSnapshots bool, registryCredentialFile string) error {
-	oneHourTimeout, _ := context.WithTimeout(context.Background(), time.Minute*60)
-	serviceAccountName := "default"
+func waitForKibishiiPods(ctx context.Context, client testClient, kibishiiNamespace string) error {
+	return waitForPods(ctx, client, kibishiiNamespace, []string{"jump-pad", "etcd0", "etcd1", "etcd2", "kibishii-deployment-0", "kibishii-deployment-1"})
+}
+
+func kibishiiPrepareBeforeBackup(client testClient, providerName string, oneHourTimeout context.Context, kibishiiNamespace, serviceAccountName, registryCredentialFile string) error {
```

Comment:
Make `context.Context` as the first parameter of this function, this is the convention.
The parameter `serviceAccountName` isn't necessary, can be a function wide variable instead

And is it possible to merge this function into the `installKibishii` which is easier to undertand?

Reply:
Yes, I updated my code as your advice both on context.Context , serviceAccountName and file organization.